mysql/tox.ini

```diff
@@ -28,4 +28,4 @@ setenv =
     8.0: MYSQL_VERSION=8.0
     maria: COMPOSE_FILE=mariadb.yaml
     maria: MYSQL_FLAVOR=mariadb
-    maria: MYSQL_VERSION=10.1.30-r1
+    maria: MYSQL_VERSION=10.1.43
```

Comment:
Note to reviewers: I had to upgrade the image version because of a bug in the RC1 we were using - it wasn't able to read the `my_custom.conf` config file properly due to a permission error.